deployment/terraform/environments/oss-vdb-test/main.tf

```diff
@@ -1,3 +1,20 @@
+locals {
+  kustomization_path = "../../../clouddeploy/gke-workers/environments/oss-vdb-test" # Path to your kustomization.yaml
+  kustomization      = yamldecode(file("${local.kustomization_path}/kustomization.yaml"))
+
+  all_resources = concat(
+    [for resource in local.kustomization.resources : "${local.kustomization_path}/${resource}" if can(regex("\\.yaml$", resource))],
+    [for patch in local.kustomization.patches : "${local.kustomization_path}/${patch.path}" if can(regex("\\.yaml$", patch.path))]
```

Comment:
Instead of reading the patches, could we read the resources from the `base/kustomization.yaml`?
(It's possible for to have a base resource without a patch)